test/unit/options.spec.ts

```diff
@@ -1204,4 +1204,74 @@ describe("Initialization", () => {
       });
     });
   });
+
+  describe("useOffset", () => {
+    it("should make viewport size integer when true", () => {
+      // Given
+      flickingInfo = createFlicking(horizontal.full, {
+        useOffset: true,
+      });
+      const flicking = flickingInfo.instance;
+      flicking.getElement().style.width = "1000.54321px";
+
+      // When
+      flicking.resize();
+
+      // Then
+      const viewportSize = (flickingInfo.instance as any).viewport.getSize();
+      expect(viewportSize).equals(parseInt(viewportSize, 10));
+    });
+
+    it("should make viewport size not integer when false", () => {
+      // Given
+      flickingInfo = createFlicking(horizontal.full, {
+        useOffset: false,
+      });
+      const flicking = flickingInfo.instance;
+      flicking.getElement().style.width = "1000.54321px";
+
+      // When
+      flicking.resize();
+
+      // Then
+      const viewportSize = (flickingInfo.instance as any).viewport.getSize();
+      expect(viewportSize).not.equals(parseInt(viewportSize, 10));
+    });
+
+    it("should make panel size integer when true", () => {
+      // Given
+      flickingInfo = createFlicking(horizontal.full, {
+        useOffset: true,
+      });
+      const flicking = flickingInfo.instance;
+      flicking.getElement().style.width = "1000.54321px";
+
+      // When
+      flicking.resize();
+
+      // Then
+      flicking.getAllPanels().forEach(panel => {
+        const panelSize = panel.getSize();
+        expect(panelSize).equals(parseInt(panelSize.toString(), 10));
+      });
+    });
+
+    it("should make panel size not integer when false", () => {
+      // Given
+      flickingInfo = createFlicking(horizontal.full, {
+        useOffset: false,
+      });
+      const flicking = flickingInfo.instance;
+      flicking.getElement().style.width = "1000.54321px";
+
+      // When
+      flicking.resize();
+
+      // Then
+      flicking.getAllPanels().forEach(panel => {
+        const panelSize = panel.getSize();
+        expect(panelSize).not.equals(parseInt(panelSize.toString(), 10));
+      });
+    });
+  });
```

Comment:
@WoodNeck I think it would be better to add test case that checks `return original panel size only if `useOffset=true` although transform is applied.`

Reply:
Okay, I'll add that.